src/library-authoring/create-library/data/api.js

```diff
@@ -0,0 +1,33 @@
+// @ts-check
+
+import { camelCaseObject, getConfig } from '@edx/frontend-platform';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+
+const getApiBaseUrl = () => getConfig().STUDIO_BASE_URL;
+
+/**
+ * Get the URL for creating a new library.
+ */
+export const getContentLibraryV2CreateApiUrl = () => `${getApiBaseUrl()}/api/libraries/v2/`;
+
+/**
+  * Create a new library.
+  * @param {import("./types.mjs").CreateContentLibraryDto} data - The library data.
+  * @returns {Promise<import("../../data/types.mjs").ContentLibrary>} The created library.
+  */
+export async function createLibraryV2(data) {
+  const client = getAuthenticatedHttpClient();
+  const url = getContentLibraryV2CreateApiUrl();
+
+  const { data: newLibrary } = await client.post(url, { ...data, description: data.title });
```

Comment:
I see that it's not in the mockups, but could you find out from Product whether or not we should be showing a `description` field?

But if not, I don't think we should populate `description` with `title` if we don't have to -- the [serializer says it allows blanks for `description`](https://github.com/openedx/edx-platform/blob/5f0ecef7fdfdc62a5278ab60ffbca150cb9c18f7/openedx/core/djangoapps/content_libraries/serializers.py#L41), so it's better to leave it blank.

Reply:
I was following the existing library-authoring form. From there, I [brought](https://github.com/openedx/frontend-app-library-authoring/blob/c7b24f7d8a2f94fd7d30017f37ba659226a08d15/src/library-authoring/create-library/data/api.js#L43C1-L48C1) the `description` populate.

But I agree that this is not a good thing. Removed from now.

Reply:
> I was following the existing library-authoring form. From there, I [brought](https://github.com/openedx/frontend-app-library-authoring/blob/c7b24f7d8a2f94fd7d30017f37ba659226a08d15/src/library-authoring/create-library/data/api.js#L43C1-L48C1) the description populate.

Hmm.. now I'm not sure what the best approach is. @bradenmacdonald could you weigh in here?

Reply:
There is a description for each library in the mockups, but we don't need to include it in the create form. I agree that leaving it blank is better than setting it to equal the title.

![Screenshot](https://github.com/open-craft/frontend-app-course-authoring/assets/945577/a4b0ba02-eb55-4955-a8c9-f90bd27d677f)